server.js

```diff
@@ -35,9 +35,10 @@ const PRODUCTION = app.get('env') === 'production';
 // overhead is included in timing.
 app.get('/ready', (req, res) => res.status(200).json({status:"ok"}));
 app.get('/live', (req, res) => res.status(200).json({status:"ok"}));
-app.get('/metrics', (req, res, next) => {
+app.get('/metrics', async (req, res, next) => {
+  const metrics = await Prometheus.register.metrics();
   res.set('Content-Type', Prometheus.register.contentType)
-  res.end(Prometheus.register.metrics())
```

Comment:
Maybe I'm reading this incorrectly - but isn't this the opposite of the suggested change?

> If your metrics server has a line like `res.send(register.metrics())`, you should change it to `res.send(await register.metrics())`.